app/views/course/assessment/submission/submissions/_submissions.html.slim

```diff
@@ -0,0 +1,12 @@
+table.table.submissions
+  thead
+    tr
+      th = t('.student_name')
+      th = t('.status')
+      th = t('.grade')
+      th = t('.experience_points')
+      th
+  tbody
+    - course_users.each do |course_user|
```

Comment:
That's one way.

Otherwise:
```slim
 - submissions = submissions.to_h { |s| [ s.course_user, s ] }
 = render partial: 'course_user_submission', collection: course_users, submissions: submissions
```
```slim
# _course_user_submission.slim
- if submissions[course_user_submission].nil?
  # no submission
 - else
  # submission
```